lua/mta_libs/wanted_constraints.lua

```diff
@@ -20,6 +20,13 @@ local ply_ents_to_remove = {
         sf._didsetoff = true
     end,
 }
+
+gameevent.Listen( "player_connect" )
+hook.Add("player_connect", "PlayerEntityLoaded", function( data )
```

Comment:
What was wrong with PlayerInitialSpawn hook?

Reply:
I'm not sure, for some reason the coder i was using was saying it was wrong. I wanted to put it in anyways but i'm not sure what the true standard is for this community rather it's "It just needs to work" or "My editor better make a smiley face".